Lib/fontTools/misc/psCharStrings.py

```diff
@@ -1043,7 +1043,8 @@ def setBytecode(self, bytecode):
 		self.program = None
 
 	def getToken(self, index,
-			len=len, byteord=byteord, isinstance=isinstance):
+			len=len, byteord=byteord, basestring=str,
+			isinstance=isinstance):
```

Comment:
You need to pass `basestring` to the `isinstance()` call below, otherwise code that depend on passing a different class will have it's behavior changed.